### ISSUE TITLE
[firefox][thunderbird] firefox-git, firefox-hg and thunderbird builds are broken

### ISSUE THREAD

**Issue by jankeromnes** [JanitorTechnology/dockerfiles#41]
I just tried building `firefox-git.dockerfile`, `firefox-hg.dockerfile` and `thunderbird.dockerfile`, and they all fail with similar `apt-get` errors:

```
Err:96 http://archive.ubuntu.com/ubuntu xenial-updates/main amd64 libpulse0 amd64 1:8.0-0ubuntu3.2
  404  Not Found [IP: 91.189.88.149 80]
Err:97 http://archive.ubuntu.com/ubuntu xenial-updates/main amd64 libpulse-mainloop-glib0 amd64 1:8.0-0ubuntu3.2
  404  Not Found [IP: 91.189.88.149 80]
Err:98 http://archive.ubuntu.com/ubuntu xenial-updates/main amd64 libpulse-dev amd64 1:8.0-0ubuntu3.2
  404  Not Found [IP: 91.189.88.149 80]
Get:99 http://archive.ubuntu.com/ubuntu xenial/main amd64 libxt-dev amd64 1:1.1.5-0ubuntu1 [394 kB]
Get:100 http://archive.ubuntu.com/ubuntu xenial/main amd64 python-dbus amd64 1.2.0-3 [83.5 kB]
Get:101 http://archive.ubuntu.com/ubuntu xenial/main amd64 python-gi amd64 3.20.0-0ubuntu1 [194 kB]
Get:102 http://archive.ubuntu.com/ubuntu xenial/universe amd64 yasm amd64 1.3.0-2 [351 kB]
Get:103 http://archive.ubuntu.com/ubuntu xenial/main amd64 libiw-dev amd64 30~pre9-8ubuntu1 [38.2 kB]
Fetched 20.4 MB in 5s (3927 kB/s)
E: Failed to fetch http://archive.ubuntu.com/ubuntu/pool/main/libe/libepoxy/libepoxy-dev_1.3.1-1ubuntu0.16.04.1_amd64.deb  404  Not Found [IP: 91.189.88.149 80]

E: Failed to fetch http://archive.ubuntu.com/ubuntu/pool/main/p/pulseaudio/libpulse0_8.0-0ubuntu3.2_amd64.deb  404  Not Found [IP: 91.189.88.149 80]

E: Failed to fetch http://archive.ubuntu.com/ubuntu/pool/main/p/pulseaudio/libpulse-mainloop-glib0_8.0-0ubuntu3.2_amd64.deb  404  Not Found [IP: 91.189.88.149 80]

E: Failed to fetch http://archive.ubuntu.com/ubuntu/pool/main/p/pulseaudio/libpulse-dev_8.0-0ubuntu3.2_amd64.deb  404  Not Found [IP: 91.189.88.149 80]

E: Unable to fetch some archives, maybe run apt-get update or try with --fix-missing?
Executing as root: sudo apt-get install -y autoconf2.13 build-essential ccache python-dev python-pip python-setuptools unzip uuid zip
Executing as root: sudo apt-get install -y libasound2-dev libcurl4-openssl-dev libdbus-1-dev libdbus-glib-1-dev libgconf2-dev libgtk-3-dev libgtk2.0-dev libiw-dev libnotify-dev libpulse-dev libx11-xcb-dev libxt-dev mesa-common-dev python-dbus xvfb yasm
Traceback (most recent call last):
  File "/tmp/bootstrap.py", line 170, in <module>
    sys.exit(main(sys.argv))
  File "/tmp/bootstrap.py", line 161, in main
    dasboot.bootstrap()
  File "/tmp/tmpAVLQ4t/mozboot/bootstrap.py", line 243, in bootstrap
    
  File "/tmp/tmpAVLQ4t/mozboot/debian.py", line 103, in install_browser_packages
  File "/tmp/tmpAVLQ4t/mozboot/debian.py", line 116, in ensure_browser_packages
  File "/tmp/tmpAVLQ4t/mozboot/base.py", line 346, in apt_install
  File "/tmp/tmpAVLQ4t/mozboot/base.py", line 302, in run_as_root
  File "/usr/lib/python2.7/subprocess.py", line 541, in check_call
    raise CalledProcessError(retcode, cmd)
subprocess.CalledProcessError: Command '[u'sudo', u'apt-get', u'install', u'-y', 'libasound2-dev', 'libcurl4-openssl-dev', 'libdbus-1-dev', 'libdbus-glib-1-dev', 'libgconf2-dev', 'libgtk-3-dev', 'libgtk2.0-dev', 'libiw-dev', 'libnotify-dev', 'libpulse-dev', 'libx11-xcb-dev', 'libxt-dev', 'mesa-common-dev', 'python-dbus', 'xvfb', 'yasm']' returned non-zero exit status 100
The command '/bin/sh -c wget -O /tmp/bootstrap.py https://hg.mozilla.org/mozilla-central/raw-file/default/python/mozboot/bin/bootstrap.py  && python /tmp/bootstrap.py --no-interactive --application-choice=browser  && rm -f /tmp/bootstrap.py' returned a non-zero code: 1
```

This looks like an `apt-get` cache problem that happens when you have a `apt-get install` or `apt-get upgrade` in a `RUN` command that has no `apt-get update` first.

**Comment by jankeromnes** [comment]
@nt1m It seems like running `sudo apt-get update -q` in the same `RUN` command as `bootstrap.py` solves this issue.

**Comment by jankeromnes** [comment]
Fixed in a7b1b583e2828744725aa5b88455d3c8d4fa3c01.
